fireblocks_sdk/sdk.py

```diff
@@ -1714,77 +2057,6 @@ def get_users(self):
 
         return self._get_request(url)
 
-    def get_users(self) -> List[Dict[str, Any]]:
-        """
-        Gets all Users for your tenant
-        """
-
-        url = "/v1/users"
-
-        return self._get_request(url)
-
-    def get_users_groups(self) -> List[Dict[str, Any]]:
-        """
-        Gets all Users Groups for your tenant
-        """
-
-        url = "/v1/users_groups"
-
-        return self._get_request(url)
-
-    def get_users_group(self, id: str) -> Dict[str, Any]:
-        """
-        Gets a Users Group by ID
-        @param id: The ID of the User
-        """
-
-        url = f"/v1/users_groups/{id}"
-
-        return self._get_request(url)
-
-    def create_user_group(self, group_name: str, member_ids: Optional[List[str]] = None) -> Dict[str, Any]:
-        """
-        Creates a new Users Group
-        @param group_name: The name of the Users Group
-        @param member_ids: The ids of the Users Group members
-        """
-
-        url = "/v1/users_groups"
-
-        body = {
-            "groupName": group_name,
-            "memberIds": member_ids
-        }
-
-        return self._post_request(url, body)
-
-    def update_user_group(self, id: str, group_name: Optional[str] = None, member_ids: Optional[List[str]] = None) -> Dict[str, Any]:
-        """
-        Updates a Users Group
-        @param id: The ID of the Users Group
-        @param group_name: The name of the Users Group
-        @param member_ids: The ids of the Users Group members
-        """
-
-        url = f"/v1/users_groups/{id}"
-
-        body = {
-            "groupName": group_name,
-            "memberIds": member_ids
-        }
-
-        return self._put_request(url, body)
-
-    def delete_user_group(self, id: str) -> None:
-        """
-        Deletes a Users Group
-        @param id: The ID of the Users Group
-        """
-
-        url = f"/v1/users_groups/{id}"
-
-        return self._delete_request(url)
-
```

Comment:
Why did you remove the `/users_groups` methods?

Reply:
It was a mistake, due to the PR existing for a long time, we should put them back

Reply:
@yanlevi @idanya Please do that as soon as possible